README.md

```diff
@@ -1,2 +1,95 @@
-# tests
-Contributed tests for Flutter
+# Flutter Tests
+
+This repository contains references to tests that are run with every
+commit to Flutter to verify that no breaking changes have been
+introduced.
+
+## Adding more tests
+
+You are welcome to add a new batch of tests. To do so, copy the file
+`registry/template.test` to create a new file in the `registry/`
+directory. Fill in the fields and delete all the comments. Then,
+submit a PR with this new file.
+
+Tests must fulfill the following criteria to be added:
+
+* All the code must be available publicly on GitHub under a license
+  compatible with this effort.
+
+* Tests must be hermetic. For example, a test should not involve
```

Comment:
What about the setup required to run a test (aside from cloning the repo)?

Reply:
like what?

Reply:
Like downloading an Android SDK or a flutter_tools plugin (when such a thing exists)

Reply:
The Android SDK should be provided by the test environment (still working on the details of that).
Plugins should be pinned by version, I guess. Maybe we should require pubspec.lock files?

Reply:
Good point - we totally should.